packages/zoe/src/contractFacet/types.js

```diff
@@ -86,8 +86,13 @@
  */
 
 /**
- * @template {object} [OR=unknown] OR is OfferResult
- * @callback MakeInvitation
+ * @typedef {<R>(
```

Comment:
It is, but I think doing that makes the generic type less readable because it's harder to scan where the generic is used in the expression. I didn't do this in https://github.com/Agoric/agoric-sdk/pull/6287 but if you feel strongly I will.

I get maybe someone would wonder "What's R stand for?" so I'm more inclined to make it `T`. It's just the generic slot.

Reply:
Second thought, the readability could be solved by the source code highlighter. It isn't now but that it could be solved downstream tips the balance for me. I made the requested change in https://github.com/Agoric/agoric-sdk/pull/6287

Reply:
> the source code highlighter

Would you say more about this tool?  Is it VSCode-only?   I use webStorm, and others here are using emacs and more obscure things.

Reply:
No particular tool; I meant whatever is highlighting the source code when one is looking at it. So WebStorm in your case and VS Code in mine. My point was that if distinguishing the generic slot is important to someone reading, they can (in principle) get their highlighter to do it.